index.js

```diff
@@ -107,6 +107,9 @@ module.exports.verify = function(jwtString, secretOrPublicKey, options, callback
     return done(new JsonWebTokenError('jwt signature is required'));
   }
 
+  if (typeof secretOrPublicKey === "undefined" || secretOrPublicKey === null) // secretOrPublicKey can be empty string
```

Comment:
@jtwebman I got impression, that key is allowed to have value of `''` (empty string).

Seems safer to check even for empty string, updated code.